pkg/receive/writer_test.go

```diff
@@ -237,12 +229,17 @@ func benchmarkWriter(b *testing.B, labelsNum int, seriesNum int) {
 	}
```

Comment:
Maybe we should check error here?

Reply:
Yes, it would be nice - no harm (little latency)

Reply:
Sorry, I forgot to commit this change it looks, I'll fix it :+1: